src/components/StatLoadingMessage.tsx

```diff
@@ -1,19 +1,14 @@
 import React from "react"
-import styled from "@emotion/styled"
-import { typography, TypographyProps } from "styled-system"
+import { Text } from "@chakra-ui/react"
 
 import Translation from "./Translation"
 
-export interface IProps extends TypographyProps {}
-
-export const IndicatorSpan = styled.span<IProps>`
-  ${typography}
-`
+export interface IProps {}
```

Comment:
This interface needs to be expanded to include a Chakra props type so that you can pass in Chakra props at the parent level. (i.e. The `Stat` component in [what-is-ethereum page](https://github.com/ethereum/ethereum-org-website/blob/f0acff0059f2eb98bbbaba29430899901dbe8e29/src/pages-conditional/what-is-ethereum.tsx#L235) currently throws a type error)

```suggestion
export interface IProps extends TextProps {}
```